webdriver/tests/bidi/browsing_context/set_viewport/device_pixel_ratio.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+from ... import get_device_pixel_ratio, get_viewport_dimensions
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("device_pixel_ratio", [1, 2])
+async def test_device_pixel_ratio_only(bidi_session, new_tab, device_pixel_ratio):
+    viewport = await get_viewport_dimensions(bidi_session, new_tab)
+
+    await bidi_session.browsing_context.set_viewport(
```

Comment:
I thought that I mentioned that already but we never actually reset the viewport and dpr, which can cause side-effects to the following tests.

Given that `set_viewport.py` is affected as well, it's not a blocker right now but we should make sure to have a follow-up for that. We might do that with our implemenation.

Reply:
Until we have resolved the discussion around https://github.com/web-platform-tests/wpt/pull/42713 we have no way to  reset the values as with this change we will never sent the null value.

Reply:
The tests are using the new_tab fixtures which I assume would create a fresh tab on every test, so we don't need to clean up explicitly?